modules/exploits/multi/http/wordpress_plugin_backup_guard_rce.rb

```diff
@@ -0,0 +1,181 @@
+##
+# This module requires Metasploit: https://metasploit.com/download
+# Current source: https://github.com/rapid7/metasploit-framework
+##
+class MetasploitModule < Msf::Exploit::Remote
+  Rank = ExcellentRanking
+
+  include Msf::Exploit::Remote::HttpClient
+  prepend Msf::Exploit::Remote::AutoCheck
+  include Msf::Exploit::CmdStager
+
+  def initialize(info = {})
+    super(
+      update_info(
+        info,
+        'Name' => 'Wordpress Plugin Backup Guard - Authenticated Remote Code Execution',
+        'Description' => %q{
+          This module allows an attacker with a privileged Wordpress account to launch a reverse shell
+          due to an arbitrary file upload vulnerability in Wordpress plugin Backup Guard < 1.6.0.
+          This is due to an incorrect check of the uploaded file extension which should be of SGBP type.
+          Then, the uploaded payload can be triggered by a call to `/wp-content/uploads/backup-guard/my_payload.php`
```

Comment:
it looks like you randomized the name, so i assume `my_payload` is no longer true

Reply:
`my_payload` was just to indicate that it was a payload file and not because it is actually the file name.